core/src/main/java/com/linecorp/armeria/common/stream/OneElementFixedStreamMessage.java

```diff
@@ -35,51 +35,72 @@ protected OneElementFixedStreamMessage(T obj) {
         this.obj = obj;
     }
 
+    @Override
+    public final boolean isEmpty() {
```

Comment:
Perhaps we can override this method in `FixedStreamMessage` and override it again in `EmptyFixedStreamMessage` to return `true`?